camera1:
  camera_name: camera1
  camera_namespace: camera
  serial_no: "746612070222"
  enable_color: true
  enable_depth: false
  enable_infra: false
  enable_infra1: false
  enable_infra2: false
  enable_sync: true
  align_depth: true
  enable_rgbd: true
  depth_module.profile: "640x480x15"
  rgb_camera.profile: "640x480x15"
  enable_compressed: true
  enable_compressed_depth: true

camera2:
  camera_name: camera2
  camera_namespace: camera
  serial_no: "944122074382"
  enable_color: true
  enable_depth: false
  enable_infra: false
  enable_infra1: false
  enable_infra2: false
  enable_sync: true
  align_depth: true
  enable_rgbd: true
  depth_module.profile: "640x480x15"
  rgb_camera.profile: "640x480x15"
  enable_compressed: true
  enable_compressed_depth: true

camera3:
  camera_name: camera3
  camera_namespace: camera
  serial_no: "944122071004"
  enable_color: true
  enable_depth: false
  enable_infra: false
  enable_infra1: false
  enable_infra2: false
  enable_sync: true
  align_depth: true
  enable_rgbd: true
  depth_module.profile: "640x480x15"
  rgb_camera.profile: "640x480x15"
  enable_compressed: true
  enable_compressed_depth: true

